workflows/differentialabundance.nf

```diff
@@ -563,15 +615,23 @@ workflow DIFFERENTIALABUNDANCE {
         params.exploratory_assay_names.split(',').collect { "${it}_matrix".toString() } +
         [ 'contrasts_file', 'versions_file', 'logo', 'css', 'citations' ]
 
+
     // Condition params reported on study type
 
-    def params_pattern = ~/^(report|study|observations|features|filtering|exploratory|differential|deseq2|gsea).*/
+    def params_pattern = "report|study|observations|features|filtering|exploratory|differential|gsea|round_digits"
```

Comment:
I think you should prefix the round digits param to place it into into one of the existing categories (maybe the report?)

Reply:
Ah yes, will do. If we add more of these params that affect multiple modules, we might consider adding a new category